clustered/client/src/main/java/org/ehcache/clustered/client/internal/EhcacheClientEntityFactory.java

```diff
@@ -91,58 +79,72 @@ public void abandonLeadership(String entityIdentifier) {
    */
   public void create(final String identifier, final ServerSideConfiguration config)
       throws EntityAlreadyExistsException, EhcacheEntityCreationException {
-    try {
-      Boolean created = asLeaderOf(identifier, new ElectionTask<Boolean>() {
-        @Override
-        public Boolean call(boolean clean) throws EntityAlreadyExistsException {
-          EntityRef<EhcacheClientEntity, UUID> ref = getEntityRef(identifier);
-          try {
-            while (true) {
-              ref.create(UUID.randomUUID());
-              EhcacheClientEntity entity = null;
+    Hold existingMaintenance = maintenanceHolds.get(identifier);
+    Hold localMaintenance = null;
+    if (existingMaintenance == null) {
+      localMaintenance = createAccessLockFor(identifier).tryWriteLock();
+    }
+    if (existingMaintenance == null && localMaintenance == null) {
+      String message = "Unable to create entity for cluster id " + identifier
+          + ": another client owns the maintenance lease";
+      LOGGER.error(message);
+      throw new EhcacheEntityCreationException(message);
```

Comment:
Especially as the code path in `DefaultClusteringService.start` will most likely print this message when `auto-create` is enabled and the entity already exists and is in use.